blackbox/docs/sql/administration/privileges.txt

```diff
@@ -0,0 +1,138 @@
+.. highlight:: psql
+.. _administration-privileges:
+
+=======================
+Cluster Wide Privileges
+=======================
+
+.. note::
+
+    Privileges are only part of the CrateDB `Enterprise Edition`_.
+    When the CrateDB Enterprise Edition is disabled, there will be
+    no user privilege checks, and every query will be executed
+    without the validation of permissions.
+
+
+``Permission Types``
+====================
+
+When a user is created, he doesn't have any privileges. The superuser can then
+grant privileges to this user.
+
+Please note that only the superuser can ``GRANT`` and ``REVOKE`` privileges.
+The superuser bypasses all privileges and can access all tables in the cluster.
```

Comment:
I would use the word permission instead of privilege now in general. 
i also agree that this paragraph is confusing. something like: Note that only the superuser can manage privileges. This user has all permissions and those cannot be managed.